react/components/ProductQuantityStepper.js

```diff
@@ -0,0 +1,80 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import { NumericStepper, withToast } from 'vtex.styleguide'
+import { orderFormConsumer } from 'vtex.store-resources/OrderFormContext'
+import { debounce } from 'lodash'
+import { findIndex, propEq } from 'ramda'
+import { injectIntl } from 'react-intl'
+
+import { productShape } from '../propTypes'
+
+class ProductQuantityStepper extends Component {
+  static propTypes = {
+    product: productShape.isRequired,
+    setUpdatingItemsState: PropTypes.func.isRequired,
+  }
+
+  constructor(props) {
+    super(props)
+    this.state = {
```

Comment:
you don't need to set this up on the constructor, you can use the same class-property syntax you're using for the other methods besides the render.

you can access the props using `this.props` as well.